variables.tf

```
@@ -5,6 +5,16 @@ variable "name" {
   default     = "dev"
 }
 
+variable "deploy_hccm" {
+  description = "Deploy hcloud-cloud-controller-manager through Helm"
+  type        = bool
+  default     = true
+}
+variable "use_cloud_routes" {
```

```suggestion
variable "hccm_use_cloud_routes" {
```

Do you think we should namespace the features flags to what they apply for?

---

This used to be `enable_hccm_routes`, but it changes not only HCCM but also the `--k3s-extra-args` and the Cilium helm chart values.